beacon-chain/rpc/eth/rewards/handlers.go

```diff
@@ -259,19 +259,27 @@ func attRewardsBalancesAndVals(
 }
 
 // idealAttRewards returns rewards for hypothetical, perfectly voting validators
-// whose effective balances are over EJECTION_BALANCE and match balances in passed in validators.
+// whose effective balances are over minIdealBalance and match balances in passed in validators.
 func idealAttRewards(
 	w http.ResponseWriter,
 	st state.BeaconState,
 	bal *precompute.Balance,
 	vals []*precompute.Validator,
 ) ([]structs.IdealAttestationReward, bool) {
 	increment := params.BeaconConfig().EffectiveBalanceIncrement / 1e9
-	maxEffectiveBalance := params.BeaconConfig().MaxEffectiveBalance / 1e9
-	ejectionBalance := params.BeaconConfig().EjectionBalance / 1e9
+	var maxEffectiveBalance, minIdealBalance uint64
+	if st.Version() < version.Electra {
+		maxEffectiveBalance = params.BeaconConfig().MinActivationBalance / 1e9
+		// Due to bail out and new penalty mechanism, validator will be exited
+		// before touching few downstairs.
+		minIdealBalance = maxEffectiveBalance - increment
+	} else {
+		maxEffectiveBalance = params.BeaconConfig().MaxEffectiveBalanceElectra / 1e9
+		// Post-Electra, the range of effective balance becomes wider, but the lower bound will be same as pre-Electra.
+		minIdealBalance = params.BeaconConfig().MinActivationBalance - increment
```

Comment:
Shouldn't params.BeaconConfig().MinActivationBalance be divided by 1e9?

Reply:
Thanks!

Reply:
Added!